objects.js

```diff
@@ -0,0 +1,84 @@
+const createPerson = (name, age) => {
+  return person = {
+    name:name,
+    age:age,
+  };
```

Comment:
Eslint is complaining here as you can use the shorthand form for this:
```
  return {
    name,
    age
  };
```
You are also assigning a variable in the return statement. Most of the time checking the eslint message will give you the answer to issues like this, its always worth reading them